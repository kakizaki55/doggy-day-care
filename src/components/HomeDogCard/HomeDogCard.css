img {
  width: 10rem;
  height: 8rem;
  border-radius: 20%;
}
.dog-card {
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  margin: 1rem;
  border: burlywood solid 5px;
  padding: 1rem;
  width: 200px;
  height: 300px;
  border-radius: 50px;
  background-color: rgba(62, 205, 131, 0.673);
}
