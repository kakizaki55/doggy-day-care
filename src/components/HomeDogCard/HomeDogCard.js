import React from 'react';
import './HomeDogCard.css';

export default function HomeDogCard({ name, age, breed, image }) {
  return (
    <>
      <h1> Meet {name} ! </h1>
      <img src={`${image}`}></img>
      <p>
        {name} is a(n) {age} year old {breed}{' '}
      </p>
    </>
  );
}
