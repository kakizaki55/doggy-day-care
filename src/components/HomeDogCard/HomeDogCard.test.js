import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';

import HomeDogCard from './HomeDogCard';
test('making sure the home dog card renders correctly', async () => {
  const container = await render(
    <BrowserRouter>
      <HomeDogCard age={11} bio="a sweet shiny boy" breed="french bull" id={11} name="happy" />
    </BrowserRouter>
  );
  expect(container).toMatchSnapshot();
});
