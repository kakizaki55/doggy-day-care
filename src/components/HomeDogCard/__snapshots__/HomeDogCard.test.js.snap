// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`making sure the home dog card renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="dog-card"
      >
        <h1>
           Meet 
          happy
           ! 
        </h1>
        <img
          src="undefined"
        />
        <p>
          happy
           is a(n) 
          11
           year old 
          french bull
           
        </p>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="dog-card"
    >
      <h1>
         Meet 
        happy
         ! 
      </h1>
      <img
        src="undefined"
      />
      <p>
        happy
         is a(n) 
        11
         year old 
        french bull
         
      </p>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
